htdocs/admin/ihm.php

```diff
@@ -330,6 +332,30 @@
 print '<td width="20">&nbsp;</td>';
 print '</tr>';
 
+// Premiere letre en majuscule
+print '<tr class="oddeven"><td class="titlefield">'.$form->textwithpicto($langs->trans("FirstToUpper"), $langs->trans("HelpFirstToUpper"), 1, 'help').'</td><td>';
+print ajax_constantonoff("MAIN_FIRST_TO_UPPER", array(), $conf->entity, 0, 0, 1, 0);
```

Comment:
The option MAIN_FIRST_TO_UPPER is "exclusive" with MAIN_ALL_TO_UPPER (if you set fist, why having option to set all or not)
Making a hidden option as a "visible feature" need to have the best practices about how these options is defined perfect.
 
So before having this becoming a stable visible feature, we must first fix the option so we have 1 and only 1 option, for example renaming the hidden option into MAIN_RULES_TO_UPPER with a value that is 'first' or 'all'